src/Infrastructure/Share/MessageBusHelper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Share;
+
+use Symfony\Component\Messenger\Exception\HandlerFailedException;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\HandledStamp;
+use Throwable;
+
+final class MessageBusHelper
```

Comment:
The idea of statics around the project it's something I don't like too much to be honest.What do you think about  have our own `CommandBus` and `QueryBus` objects injecting the required MessageBusInterface and handling the differences between one and the other independently (returning result and not). A Trait for the exception handling part should be enough to be DRY.
How does it sounds to you?

Reply:
Sounds great for me, let's do this!